backend/src/main/java/com/hyooun/wilson/review/domain/ReviewResult.java

```diff
@@ -0,0 +1,14 @@
+package com.hyooun.wilson.review.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+
+@Getter
```

Comment:
ReviewResult 클래스에 생성자 추가하세요.

```suggestion
public ReviewResult(String summary, List<String> suggestions) {
	this.summary = summary;
	this.suggestions = suggestions;
}
```